src/toil/test/src/jobCacheTest.py

```diff
@@ -252,9 +252,9 @@ def _testCacheEviction(self, file1MB, file2MB, diskRequestMB):
                 Job.Runner.startToil(A, self.options)
             except FailedJobsException as err:
                 self.assertEqual(err.numberOfFailedJobs, 1)
-                errType, errMsg = self._parseAssertionError(self.options.logFile)
-                if (errType == 'AssertionError' and
-                        errMsg == 'Unable to free up enough space for caching.'):
+                with open(self.options.logFile) as f:
+                    logContents = f.read()
+                if 'AssertionError: Unable to free up enough space for caching.' in logContents:
```

Comment:
This is just as brittle, no? It's still duplicating a hard-coded error message. There must have been a reason for the intelligent parsing done by `_parseAssertionError`.